packit_service/service/api/usage.py

```diff
@@ -0,0 +1,543 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from http import HTTPStatus
+from logging import getLogger
+from typing import Any
+
+from flask import request
+from flask_restx import Namespace, Resource
+
+from packit_service.models import (
+    CoprBuildTargetModel,
+    GitProjectModel,
+    JobTriggerModelType,
+    KojiBuildTargetModel,
+    SRPMBuildModel,
+    SyncReleaseModel,
+    TFTTestRunTargetModel,
+    VMImageBuildTargetModel,
+)
+from packit_service.service.api.utils import response_maker
+
+logger = getLogger("packit_service")
+
+usage_ns = Namespace("usage", description="Data about Packit usage")
+
+
+@usage_ns.route("")
+class Usage(Resource):
+    @usage_ns.response(HTTPStatus.OK, "Providing data about Packit usage")
+    def get(self):
+        """List all SRPM builds."""
+
+        top = int(request.args.get("top"))
+        datetime_from = request.args.get("from")
+        datetime_to = request.args.get("to")
```

Comment:
will these be set/hardcoded in the dashboard?

Reply:
I am thinking about showing (some) data from the last month and year.

Maybe, I can create another endpoint to prepare the expected data and avoid code on the dashboard side.

Reply:
I'm in favor of having that code (=defaults) in dashboard, but no strong opinion.